main.cpp

```diff
@@ -12,38 +12,59 @@
 
 using namespace std;
 
-string value[13] = {"1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "Jack", "Queen", "King"};
-string suit[4] = {"Hearts", "Diamonds", "Clubs", "Spades"};
-
-class Card {
-public:
+class Card
+{
+private: // Made value and suit private. -Gage Miller
     string value;
     string suit;
+
+public:
+    Card() : value(""), suit(""){}; // Default constructor
+    Card(string value, string suit) : value(value), suit(suit){};
+
+    // Created getters for value and suit. -Gage Miller
+    string getValue()
+    {
+        return value;
+    }
+
+    string getSuit()
+    {
+        return suit;
+    }
 };
 
-class Deck {
+class Deck
+{
 public:
     vector<Card> sortedCards;
     vector<Card> shuffledCards;
 
-    void createDeck() {
+    void createDeck()
+    {
+        // I moved the value and suit strings here, rather than keeping them global. -Gage Miller
+        string value[13] = {"1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "Jack", "Queen", "King"};
+        string suit[4] = {"Hearts", "Diamonds", "Clubs", "Spades"};
+
         // concatenate two arrays of value + suit together
-        for(const auto & x : value) {
-            for (const auto & y : suit) {
-                Card card;
-                card.value = x;
-                card.suit = y;
+        for (const auto &x : value)
+        {
+            for (const auto &y : suit)
+            {
+                Card card = Card(x, y);
                 sortedCards.push_back(card);
             }
```

Comment:
@miller2590 I agree, this is more concise and better OOP with the getters and setters.